code/modules/events/intel_computer.dm

```diff
@@ -0,0 +1,16 @@
+/datum/round_event_control/intel_computer
+	name = "Intel computer activation"
+	typepath = /datum/round_event/intel_computer
+	weight = 25
+
+/datum/round_event/intel_computer/start()
```

Comment:
Should there be a can_start proc?

For example what happens if all the intel computers got removed by an admin.

Reply:
Its handled on start where it just doesnt do anything if the prerequisites are not fulfilled

Reply:
But i dont believe there is anyway for an event itself to look at the current state of the game and choose not to run.
For example if there was an event for any survivors but if the current game had none (either dead or none spawned).

It couldn't do that dynamically at runtime.